src/Compiler/Driver/fsc.fs

```diff
@@ -689,6 +689,22 @@ let main1
     AbortOnError(diagnosticsLogger, exiter)
     ReportTime tcConfig "Typechecked"
 
+    // Fixup typars
+    let rec kindaEvilFixingEntity (entity: Entity) =
```

Comment:
Can this be named in regular manner, e.g. `fixupTyparsNamesInEntity` and `fixupTyparNames` or similar, and comment be used instead please?